.ci-operator.yaml

```diff
@@ -2,4 +2,4 @@
 build_root_image:
```

Comment:
Do we also need to update `go.mod` file? i.e.
```
go mod edit -go=1.17
```

Reply:
Yes, good catch